runtime/android/core_internal/src/org/xwalk/core/internal/SwipeRefreshHandler.java

```diff
@@ -0,0 +1,185 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Copyright (c) 2016 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package org.xwalk.core.internal;
+
+import android.content.Context;
+import android.view.ViewGroup.LayoutParams;
+
+import org.chromium.base.annotations.JNINamespace;
+import org.chromium.base.metrics.RecordUserAction;
+import org.chromium.base.TraceEvent;
```

Comment:
So， the original pull is correct? https://github.com/axinging/crosswalk/commit/3f21e93cf6487994379c1af917b35edfd1fac22d

Again, I hope we do not change this file too much. This is just a copy.